compiler/rustc_middle/src/ty/print/pretty.rs

```diff
@@ -817,148 +817,168 @@ pub trait PrettyPrinter<'tcx>:
             }
         }
 
+        {
```

Comment:
literally have no idea what i'm doing with `define_scoped_cx!()` but this is the only way for me to get this to work

Reply:
hmm... what problems did you have when you kept the previous single call to `define_scoped_cx!`?